templates/jlink/jlink-builder/jdk-11-buildconfig.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata: 
+  name: jlink-builder-jdk-11
```

Comment:
The JDK version will probably eventually be a template parameter for an OpenShift template. If keeping the three buildconfig YAMLs in sync proves to be a burden, consider deleting two of them (assuming there's no substantive difference besides JDK version) and we can re-introduce them once the content stabilises. Your call!